synonymdev/bitkit-react-native
# feat(send): update Send flow

### Description

Updates send flow for design prototype v42

### Linked Issues/Tasks

https://app.asana.com/0/1189635884316057/1205408884225234/f

### Type of change


### Tests


### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/2b052b68-e371-4176-91b0-da94ff7046f1

### QA Notes

Make sure send transaction information is kept and updated as expected when navigating back and forth or switching to scanner.
